tests/integration/targets/nxos_ospfv3/tests/common/parsed.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- debug:
+    msg: START nxos_ospfv3 parsed integration tests on connection={{ ansible_connection }}
+
+- name: Parse externally provided OSPFv3 config
+  register: result
+  cisco.nxos.nxos_ospfv3:
+    running_config: "{{ lookup('file', './fixtures/parsed.cfg') }}"
+    state: parsed
+
+- name: Assert that config was correctly parsed
+  assert:
+    that:
+      - "{{ merged['after']['processes'] | symmetric_difference(result['parsed']['processes']) |length == 0 }}"
```

Comment:
Can we have an assertion for all non-action states, to check the value of result['changed']?

Reply:
This is also handled by the Engine, but since that is not tested, I'll add it here.

Reply:
Oh, actually, we did cover this for gathered and parsed in the unit tests. Only missing one was rendered.

Reply:
Fixed.